README.md

```diff
@@ -1,290 +1,5 @@
-
-## Setup 
-1. Das Repository downloaden, clonen, oder die Template Funktion nutzen, um ein eigenes Repository zu erstellen.
-1. Mit der Update-Datei (OS abhängig .bat/.sh) kann das Template aktualisiert werden
-* ### **ACHTUNG** die Update-Datei (entsprechend OS) 
-  * **überschreibt** die Dateien im build Ordner
-    * eigene Anpassungen sollten über die Dateien im Root-Ordner geschehen
-  * **löscht** die mit git in Zusammenhang stehenden Ordner und Dateien (.git, .gitignore, .gitkeep)
-
-Das Projekt sollte sich nun bauen lassen.
-
-Wir empfehlen den `latexmk` Befehl zum kompilieren des Projekts (erfordert eine Perl Installation).
-
-
-# Latex Tipps
-
-<!-- todo: add more -->
-Ein relativ simples [LaTeX-Tutorial](https://www.latex-tutorial.com/tutorials/first-document/) zum einfachen Einstieg in die Welt von TeX. 
-> _ist allerdings durch das Template nicht nötig, nachfolgendes sollte ausreichen_
-
-## "Variablen"
-Variablen gibt es in TeX an sich nicht wie in anderen Sprachen.
-```
-\def\<variablenName>{<variablenWert>}
-```
-Der `\def` Befehl definiert ein Command der letztendlich dem folgendem entspricht
-```
-\newcommand{\<variablenName>}{<variablenWert>}
-```
-Richtig werden diese "Variablen" dann durch `\<variablenName>`**`{}`** aufgerufen.
-
-Es ist aber auch möglich sie einfach nur durch `\<variablenName>` aufzurufen.\
-> Hier ist zu beachten ist das nach der Variable das Leerzeichen fehlt wird, da dieses als Argument aufgenommen wird
-
-## Environment (Umgebung)
-Ein Codeblock welcher bestimmte Anläufe vor und nach dem eigenen Code laufen lässt.
-```
-\begin{<environment>}
-  <codeAndText>
-\end{<environment>}
-```
-### Common Environment's
-* [itemize/enumerate](https://en.wikibooks.org/wiki/LaTeX/List_Structures)
-* [table/tabular](https://en.wikibooks.org/wiki/LaTeX/Tables)
-* [figure](https://en.wikibooks.org/wiki/LaTeX/Floats,_Figures_and_Captions)
-
-# Zitate und Literaturverzeichnis
-
-## Zitat als Fußnote einfügen
-
-Dafür wird der `footcite` Befehl genutzt. Dieser bietet folgende Syntax: 
-```
-\footcite[Postnote]{literatur_id}
-```
-
-### Beispiel
-```
-\footcite[S. 42]{mapi}
-```
-
-## Hochgestelltes Zitat einfügen
-Alternativ kann nun auch der `supercite` Befehl verwendet werden:
-```
-\supercite[Postnote]{literatur_id}
-```
-
-### Beispiel
-```
-\supercite[S. 42]{mapi}
-```
-
-***
-
-<!-- todo: is this still a thing? the vscode extension does this pretty well without crying -->
-**Bei jeder Änderung in `literatur.bib` müssen folgende Schritte durchgeführt werden:**
-1. Das Projekt kompilieren (`pdflatex.exe -synctex=1 -interaction=nonstopmode "template".tex`)
-1. Biber ausführen (`biber.exe "template"`)
-1. Das Projekt 2x kompilieren
-***
-> Werden die oben genannten Schritte nicht durchgeführt, kommt es zu Darstellungsfehlern bei Zitaten und dem Literaturverzeichnis.
```

Comment:
Yes, this is absolutely still a thing in TeXstudio!

I have to:

- ``\footcite`` my source
- Compile, resulting in a broken reference
- run ``biber``
- compile again - now everything is OK

Reply:
Oh rip, ja in vscode kümmert sich latexmk drum.

Reply:
Na fürs nächste Mal probier ich vscode aus xD

Reply:
Ich muss leider etwas meckern xD da du kein vscode nutzt hast du die indent size angepasst (⌒_⌒;)
In der [.editorconfig](https://github.com/RvNovae/dhge-latex/blob/main/.editorconfig) Datei ist das definiert, weshalb es evtl. gut gewesen wäre hättest du code genutzt ;)

Nichtsdestotrotz, vielen Danke fürs fixen <3

Reply:
Hmm ich habe aber VScode benutzt. Aber ja, ich habe nicht gesehen, dass die Indent Size definiert ist; ich habe nur gelbe Linter-Linien gesehen 😅 

Nächstes Mal prüfe ich das gründlicher :)

Reply:
Was für eine Linter hast du den eigentlich genutzt?

Ich erstell noch mal ein Issue für das folgende:
Und evtl. sollte ich nochmal in die Doku schreiben das bei VSCode Nutzung zu beachten ist das .editorconfig-extension zu installieren... weil leider ist das nicht Default mäßig installiert (wie z.B. in Intellij hehe)

Wenn du in den Extensions nach `@recommended` suchst findest du dann auch die recommended extensions wie z.B. die für  .editorconfig

![image](https://user-images.githubusercontent.com/36958734/125598611-086a7491-b701-4ca5-9d48-8c6b599c55ba.png)

Reply:
ich hab ``markdownlint`` von David Anson (``davidanson.vscode-markdownlint``)

Reply:
häng dann mal bitte die Linter Extension an #92
danke^^